src/com/android/calendar/selectcalendars/SelectVisibleCalendarsFragment.java

```diff
@@ -39,6 +39,9 @@
 
 import ws.xsoh.etar.R;
 
+/**
+ * TODO: This fragment is still used in the tablet design
```

Comment:
do you still want to do this? Did you test it on a tablet?

Reply:
Yep, I tested it on a tablet.
I would leave this fragment in Etar for now to not destroy the tablet design. But in the long run this could be replace with a different implementation.